docs/v3.x/concepts/queries.md

```diff
@@ -363,12 +363,11 @@ Documentation: [https://mongoosejs.com/](https://mongoosejs.com/)
 
 **Example**
 
-```js
-const result = strapi.query('restaurant').model.find({
+```js 
+const query = strapi.query('restaurant').model.find({
   date: { $gte: '2019-01-01T00.00.00Z' },
-});
-
-const fields = result.toObject();
+}).lean();
```

Comment:
Using lean will not apply / add virtual fields nor data formatting. You should not be using it in this example.